src/test/scala/edu/berkeley/ce/rockslicing/InputProcessingSpec.scala

```diff
@@ -5,169 +5,267 @@ import org.scalatest.FunSuite
 import scala.io.Source
 
 class InputProcessingSpec extends FunSuite {
+
+  def compareJointSets(jointSet1: JointSet, jointSet2: JointSet): Boolean = {
+    if (jointSet1.strike == jointSet2.strike &&
+      jointSet1.dip == jointSet2.dip &&
+      jointSet1.jointSpacing == jointSet2.jointSpacing &&
+      jointSet1.persistence == jointSet2.persistence &&
+      jointSet1.stochasticFlag == jointSet2.stochasticFlag &&
+      jointSet1.strikeStDev == jointSet2.strikeStDev &&
+      jointSet1.dipStDev == jointSet2.dipStDev &&
+      jointSet1.jointSpacingStDev == jointSet2.jointSpacingStDev &&
+      jointSet1.persistenceStDev == jointSet2.persistenceStDev
+    ) {
+      true
+    } else {
+      false
+    }
+  }
+
   test("Input file without global origin specified should produce error") {
-    val inputStr = "-1.0 0.0 0.0 0.0 0.0 0.0\n" +
-      "1.0 0.0 0.0 2.0 0.0 0.0\n" +
-      "0.0 -1.0 0.0 0.0 0.0 0.0\n" +
-      "0.0 1.0 0.0 2.0 0.0 0.0\n" +
-      "0.0 0.0 -1.0 0.0 0.0 0.0\n" +
-      "0.0 0.0 1.0 2.0 0.0 0.0\n" +
-      "1.0 0.0 0.0 0.0 0.0 0.0 1.0 0.0 0.0 0.0 0.0\n"
+    val inputStr = "0.0 0.0 0.0 2.0 2.0 2.0\n" +
```

Comment:
We use that approach in `EndToEndSpec.scala` for the input file. In this case, I think it's easier to keep these in place here: If we change the code, now we have to go back and figure out which input file corresponds to what unit test in `InputProcessingSpec` as opposed to just having it right there in front of you while trying to change/debug things.

Do you agree with that?